.github/workflows/merge.yml

```diff
@@ -22,10 +22,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up JDK 8
+    - name: Set up JDK 11
```

Comment:
We still need to ensure that this works for java 8 given internal workloads which haven't moved over. I think I might have also fat fingered the contrib docs on this recently, which I'll go check.

Reply:
Mmmh, I was following the README.md, where it states, the lib requires JDK11+. Hence I used language features (e.g. `List.of`, `Map.of`, `var`) that is supported by this minimum version. To make it clear. This PR would make the lib incompatible with Java 8 clients. If JDK8 needs to be supported then I will need to adapt the PR. 
From my perspective, it would be beneficial to base this lib on Java 11, even though we might loose some users. Java 11 is out for more than 3 years now and is considered outdated itself. Java 8 is 8 years old at this point and Oracle ended its commercial support in March '22. Having more libs being incompatible w/ such old versions might increase the pressure to update to new JDK versions. IMHO, this would be favorable for us, as maintenance is easier and for the whole industry due to security patches and updates.

Please let me know, which direction we want to go.

Reply:
I'm very much on your side with "How are folks not on Java 11 yet?!". The reality is there are several internal usecases to eBay that are still on JDK8. Given I'm authoring the library I get to (at least for now) push my pet use-case of our internal needs and support JDK 8.

I'll fix the README. I agree that it would be beneficial to be on JDK11, but I don't want to do that just yet. Sorry for the confusion!

Reply:
I know from personal experience that there's a unfortunate amount of JDK8 out there. I won't comment on if that's the case where I work. :sweat_smile: